README.md

```diff
@@ -9,14 +9,36 @@ that are battle-tested in [Servo](https://servo.org/), split in two crates:
 Mozjs is currently tracking SpiderMonkey on [ESR-115](https://searchfox.org/mozilla-esr115/source/) branch
 (currently version 115.3).
 
-## Building
+## Building from Pre-built Archive
+
+SpiderMonkey is very large which could take a long time to compile. If building with default
+feature, `mozjs` provides a pre-built archive which can be linked against. You can also create
+your own archive and link to it. `mozjs` currently offers two environment variables to enable
+this feature:
+
+- `MOZJS_CREATE_ARCHIVE=1` will create a SpiderMonkey binary tarball for release usage. It will
+   be created in the `target` directory.
```

Comment:
```suggestion
- `MOZJS_CREATE_ARCHIVE=1` will create a SpiderMonkey binary archive for release usage. It will
   be created in the `target` directory.
```